src/front/glsl_new/parser.rs

```diff
@@ -41,7 +41,24 @@ pomelo! {
     %type function_definition Function;
 
     root ::= version_pragma translation_unit;
-    version_pragma ::= Version IntConstant Identifier?;
+    version_pragma ::= Version IntConstant(V) Identifier?(P) {
+        match V.1 {
+            450 => (),
```

Comment:
Can we support at least 440 as well?
When going through glslang, 450 is annoying since it inserts implicit `gl_CullDistance` output, which isn't supported by Naga or WGSL today. This isn't a problem for your code, but if we are to test it on the same shaders versus glslang, it would be nice to support 440

Reply:
I don't really have any strong feelings for 450, just wanna start with a single version, so maybe 440 would be better starting point :)

Reply:
can we accept both versions, but for now treat them equally?

Reply:
sure :)

Reply:
maybe include 460 as well then?

Reply:
sure